SampleComponentLibraryApp/SampleComponentLibraryApp/HeaderView/HeaderViewViewController.swift

```diff
@@ -0,0 +1,79 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+class HeaderViewViewController: UIViewController, Themeable {
+    let headerTitle = "Website Title"
+    let headerSubtitle = "Website Subtitle"
+    let errorHeaderSubtitle = "Website Error"
+
+    var themeManager: ThemeManager
+    var themeObserver: NSObjectProtocol?
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+
+    private lazy var headerView: HeaderView = .build()
+    private lazy var errorHeaderView: HeaderView = .build()
+
+    init(themeManager: ThemeManager = AppContainer.shared.resolve()) {
+        self.themeManager = themeManager
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+
+        listenForThemeChange(view)
+        applyTheme()
+
+        headerView.adjustLayout()
+        errorHeaderView.adjustLayout()
+
+        headerView.setupDetails(subtitle: headerSubtitle,
+                                title: headerTitle,
+                                icon: UIImage(named: "faviconFox"))
+        errorHeaderView.setupDetails(subtitle: errorHeaderSubtitle,
+                                     title: headerTitle,
+                                     icon: UIImage(named: "faviconFox"),
+                                     warningIcon: "criticalFillLarge",
```

Comment:
to maintain consistency with our other files, what do you think about updating this strings to be pulled from `StandardImageIdentifiers`